parm/post_avblflds.xml

```diff
@@ -6312,14 +6330,15 @@
 
     <param>
        <post_avblfldidx>687</post_avblfldidx>
-       <shortname>NITRATE_AER_OPT_DEP_at550</shortname>
+       <shortname>NITRATE_AER_SCAT_OPT_DEP_at550</shortname>
```

Comment:
@zhanglikate Would you want to update the postcntrl_ufs_aerosol*xml files to reflect this short name change?

Reply:
NITRATE_AER_OPT_DEP_at550 has been moved to 615